components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/services/TemplateManagerService.java

```diff
@@ -196,15 +228,21 @@ public void deleteBusinessRule(String uuid) throws TemplateManagerException { //
                     removeBusinessRuleDefinition(uuid);
                 } catch (TemplateManagerException e) {
                     log.error("Failed to delete Business Rule definition of : " + uuid, e);
+                } catch (SQLException e) {
+                    e.printStackTrace();
```

Comment:
Remove and add log.error with context

Reply:
@AnuGayan Please prioritize the fixes like this and add relevant commit ID when you reply.

Reply:
Fixed by 47e9832228beaa5747e074303ac9d729cc69abdf